### ISSUE TITLE
PCA visualizer is not rendering 3D plot

### ISSUE THREAD

**Issue by Kautumn06** [DistrictDataLabs/yellowbrick#984]
The `PCADecomposition` visualizer should generate a 3D plot when passed `proj_dim=3`; however, it currently needs to be explicitly passed a `matplotlib` 3D axis in order to render correctly. For instance, the following code will return a 2D plot despite the `proj_dim` parameter being set to `3`.
```
from yellowbrick.datasets import load_credit
from yellowbrick.features import PCADecomposition

X, y = load_credit()

visualizer = PCADecomposition(scale=True, proj_dim=3)
visualizer.fit_transform(X, y)
visualizer.show()
```
![pca-2](https://user-images.githubusercontent.com/19958803/67163645-52d9b780-f33f-11e9-877e-bf524948d778.png)

However, a 3D plot will be returned when explicitly passed a 3D axis, as in this example:
```
import matplotlib.pyplot as plt
from mpl_toolkits.mplot3d import Axes3D

from yellowbrick.datasets import load_credit
from yellowbrick.features import PCADecomposition

X, y = load_credit()

# Create a figure and add a 3D axis
fig = plt.figure()
ax = fig.add_subplot(111, projection='3d')

visualizer = PCADecomposition(scale=True, proj_dim=3, ax=ax)
visualizer.fit_transform(X, y)
visualizer.show()
```
![pca-3D](https://user-images.githubusercontent.com/19958803/67163679-dc898500-f33f-11e9-9e38-80de321ccf4d.png)

**Comment by naresh-bachwani** [comment]
Hey there @Kautumn06,
We changed the `proj_dim` parameter to `projection` in #908. Try using projection! :)

**Comment by pdamodaran** [comment]
Thanks @naresh-bachwani for pointing this out!  I tested this out using the new parameter and I can now see a 3d plot.

```
from yellowbrick.datasets import load_credit
from yellowbrick.features import PCADecomposition

X, y = load_credit()

visualizer = PCADecomposition(scale=True, projection=3)
visualizer.fit_transform(X, y)
visualizer.show()
```

![image](https://user-images.githubusercontent.com/7106134/69917441-2338c780-1434-11ea-8ba2-28be0d0a4d60.png)
